packages/mint-components/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/mint-components",
-  "version": "1.1.1",
+  "version": "1.1.2-12",
```

Need to update to `1.1.2`.

```suggestion
  "version": "1.1.2-12",
```